src/NLog.Web/README.md

```diff
@@ -0,0 +1,14 @@
+# NLog.Web
+
+- LoggerProvider for ASP.NET platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
+
+.NET 3-5-4.8
+
```

Comment:
With NLog v5 then it is very important to manually "register" NLog.Web in NLog.config, think this should be included:
```xml
  <!-- enable ASP.NET layout renderers -->
  <extensions>
    <add assembly="NLog.Web"/>
  </extensions>
```

Reply:
Complete for both README.md files